src/xdocs/config_filters.xml

```diff
@@ -973,6 +973,50 @@ public class InputTest {
 &lt;suppress-xpath checks=&quot;.*&quot; query=&quot;//METHOD_DEF[@text='legacyMethod']
           /descendant-or-self::node()&quot;/&gt;
         </source>
+        <p>
+          Currently, xpath queries support two types of attributes <code>@text</code> and
+          <code>@value</code>
+          <ul>
+            <li>
+              <code>@text</code> - addresses to the name of the node. For example: class name,
+              method name, variable name and etc.
+            </li>
+            <li>
+              <code>@value</code> - addresses to the value of the node. For example: variable value,
+              annotation value, text content and etc.
+            </li>
+          </ul>
+        </p>
+        <p>For instance, variable <code>wordCount</code> must be suppressed by xpath query.</p>
+        <source>
+public class InputTest {
+    private int wordCount = 11;
+}
+        </source>
+        <p>The easiest way is to suppress by variable name:</p>
+        <source>
+&lt;suppress-xpath checks=&quot;.&quot; query=&quot;//VARIABLE_DEF[@text='wordCount']&quot;/&gt;
+        </source>
+        <p>Another way is to suppress by variable value:</p>
```

Comment:
@romani the link is http://rveach.no-ip.org/checkstyle/checkstyle.php . Link and documentation can also be found at https://github.com/checkstyle/contribution/tree/master/checkstyle-web-tester , so it is already public knowledge.

You may add it to checkstyle if you wish, just know if I ever lose internet in a storm it will go down too and it is my private server.

Reply:
done